File: AppLoginJavaBiding/app/src/main/java/com/paulo/apploginjavabiding/MainActivity.java
package com.paulo.apploginjavabiding;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import com.paulo.apploginjavabiding.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());


        binding.btnEntrar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String login = binding.edtLogin.getText().toString();
                String senha = binding.edtSenha.getText().toString();


                if(login.equals("PAULO") && senha.equals("1234")){
                    Toast.makeText(getApplicationContext(), "Senha Correta", Toast.LENGTH_SHORT).show();
                }else{
                    Toast.makeText(getApplicationContext(), "Senha Incorreta", Toast.LENGTH_SHORT).show();

                }

            }
        });


    }
}

File: ConversorTemperatura/app/src/main/java/com/paulo/conversortemperatura/MainActivity.kt
package com.paulo.conversortemperatura

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.paulo.conversortemperatura.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    private lateinit var binding: AppCompatActivity
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}